hugegraph-api/src/main/java/com/baidu/hugegraph/api/gremlin/CypherAPI.java

```diff
@@ -0,0 +1,93 @@
+package com.baidu.hugegraph.api.gremlin;
+
+import org.opencypher.gremlin.translation.TranslationFacade;
+import org.slf4j.Logger;
+
+import com.baidu.hugegraph.api.filter.CompressInterceptor;
+import com.baidu.hugegraph.util.E;
+import com.baidu.hugegraph.util.Log;
+import com.codahale.metrics.annotation.Timed;
+
+import jakarta.inject.Singleton;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.Response;
+
+@Path("graphs/{graph}/cypher")
```

Comment:
could you please also bump up api version and adapt java-client:
https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-api/src/main/java/com/baidu/hugegraph/version/ApiVersion.java#L124

Reply:
see https://github.com/apache/incubator-hugegraph-toolchain/pull/272